grass7/raster/r.centroids/r.centroids.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+############################################################################
+#
+# MODULE:    r.centroids
+#
+# AUTHOR(S): Caitlin Haedrich (caitlin dot haedrich gmail com) 
+#
+# PURPOSE:   Wrapper for r.volume. Creates vector map of centroids from a 
+#            raster of "clumps"; r.clumps creates "clumps" of data.
+#
+# COPYRIGHT: (C) 2021 by Caitlin Haedrich and the GRASS Development Team
+#
+#            This program is free software under the GNU General Public
+#            License (>=v2). Read the file COPYING that comes with GRASS
+#            for details.
+#
+#############################################################################
+
+#%module
+#% description: Creates vector map of centroids from raster of "clumps".
+#% keyword: raster
+#% keyword: centroid
+#% keyword: clumps
+#% keyword: vector
+#% keyword: centerpoint
+#%end
+
+#%option G_OPT_R_INPUT
+#% key: input
```

Comment:
`#% key: output` and `#% required: yes`: these are the defaults for G_OPT_R_INPUT, so not needed. Same for G_OPT_V_OUTPUT. Just trying that might be more understandable, but anyway here the definitions of these so called standard options:

https://grass.osgeo.org/grass78/manuals/parser_standard_options.html

Reply:
Deleted the both lines in the G_OPT_R_INPUT and G_OPT_V_OUTPUT.